Core/src/org/sleuthkit/autopsy/casemodule/services/TagsManager.java

```diff
@@ -224,11 +237,36 @@ public synchronized TagName addTagName(String displayName, String description) t
      *                                       name to the case database.
      */
     public synchronized TagName addTagName(String displayName, String description, TagName.HTML_COLOR color) throws TagNameAlreadyExistsException, TskCoreException {
+        TskData.FileKnown knownStatus = TskData.FileKnown.UNKNOWN;
+        if (getNotableTagDisplayNames().contains(displayName)) {
+            knownStatus = TskData.FileKnown.BAD;
+        }
+        return addTagName(displayName, description, color, knownStatus);
```

Comment:
Let's just go ahead and pass unknown to the four args method, I think it will be a tad simpler.